src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+body > div {
+  margin: auto;
+  width: 300px;
+  height: 100px;
+}
+
+body > div:hover {
+  height: 200px;
+}
```

Comment:
Try to avoid increasing the specificity of the selector. One class is enough to identify the elements on the page. No need to write something like .block.colored-block. Use .block and .block--colored instead, and apply each of them to your element